fenix/app/src/main/java/org/mozilla/fenix/settings/logins/fragment/SavedLoginsAuthFragment.kt

```diff
@@ -54,6 +55,11 @@ class SavedLoginsAuthFragment : PreferenceFragmentCompat() {
         startForResult = registerForActivityResult {
             navigateToSavedLoginsFragment()
         }
+
+        val args by navArgs<SavedLoginsAuthFragmentArgs>()
+        if (args.openedFromHomeShortcut) {
+            verifyCredentialsOrShowSetupWarning(requireContext())
```

Comment:
Could we add a comment here like:

// Handle auth and then navigate to saved logins

I think the `verifyCredentialsOrShowSetupWarning` could use a more clear name, but that's outside the scope of this patch.